Add Attribute Types to NFTs pallet (#4905)
w3f/polkadot-wiki: docs/learn/learn-nft-pallets.md

collection is created and all the items are minted. Now, by setting roles to `none` we remove the
possibility of minting any more items, changing the metadata, or disallowing some item's transfer.

### Attributes

> A collection can hold the following types of attributes:

Reviewer: ```suggestion
An item can hold the following types of attributes:
```
